FirebaseInstallations/Source/Library/InstallationsStore/FIRInstallationsStoredAuthToken.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Add short docs to the API
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSInteger, FIRInstallationsAuthTokenStatus) {
+  // An initial status or an undefined value.
+  FIRInstallationsAuthTokenStatusUnknown,
+  FIRInstallationsAuthTokenRequestInProgress,
+  FIRInstallationsAuthTokenStatusTokenReceived
+};
+
```

Comment:
What is this class for?  Should the information in installationItem include this as well?

Reply:
This class is used for `FIRInstallationsItem.authToken`